src/components/auth/auth.service.ts

```diff
@@ -184,6 +188,93 @@ export class AuthService {
     return session;
   }
 
+  async forget(input: UserEmailInput): Promise<boolean> {
```

Comment:
```suggestion
  async forgotPassword(email: string): Promise<void> {
```
There's only ever one parameter needed here, `email`, so I would avoid creating an input type for it and just have the argument directly on the query.

We also don't want to return booleans that indicate success. These either work or throw an error. GraphQL doesn't have the concept of a `void` return so in those cases we `return true` in the resolver. But that's a graphql transport detail which is why it goes there and not in the service.